### DISCUSSION TITLE
Not able to build the firefox-ios for MacOS 14 with Xcode 15 version

### DISCUSSION THREAD

**Discussion by pchandak1408** [mozilla-mobile/firefox-ios#16563] [General]
Do we have support to build firefox-ios with Xcode15 on macOS 14? I am trying to build but not able to procced successfully.

**Comment by lmarceau** [comment]
Hello! There are known issues that are being worked on to support Xcode 15. I see we have at least issues https://github.com/mozilla-mobile/firefox-ios/issues/16476 and https://github.com/mozilla-mobile/firefox-ios/issues/16549, there might be more to come to enable Xcode 15 on MacOS 14.

**Comment by adudenamedruby** [comment]
This should have all been resolved by now, as all the devs are on macOS 14.x and xcode 15
